web/src/cypress/support/alert.ts

```diff
@@ -47,6 +47,59 @@ declare global {
   }
 }
 
+function getAlertLogs(id: number): Cypress.Chainable<Array<AlertLog>> {
+  const query = `query GetLogs($id: Int!, $after: String!) {
+    alert(id: $id) {
+      recentEvents(input:{limit:149, after: $after}) {
+        nodes {
+          timestamp
+          message
+        }
+        pageInfo {
+          hasNextPage
+          endCursor
+        }
+      }
+    }
+  }`
+
+  // NOTE next recursively builds logs to ultimately yield an AlertLog[]
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const next = (logs: AlertLog[], after: string): Cypress.Chainable<any> =>
+    cy.graphql2(query, { id, after }).then(res => {
+      if (res.alert.recentEvents.pageInfo.hasNextPage) {
+        return next(
+          logs.concat(res.alert.recentEvents.nodes as Array<AlertLog>),
+          res.alert.recentEvents.pageInfo.endCursor,
+        )
+      }
+
+      return logs.concat(res.alert.recentEvents.nodes)
+    })
+
+  return next([], '')
+}
+
+function getAlert(id: number): Cypress.Chainable<Alert> {
+  const query = `query GetAlert($id: Int!) {
+    alert(id: $id) {
+      number: _id, id, summary, details, serviceID: service_id,
+      service {
+        id, name, description
+        epID: escalation_policy_id,
+        ep: escalation_policy {
+            id
+            name
+            description
+            repeat
+        }
+      }
+    }
+  }`
+
+  return cy.graphql(query, { id }).then(res => res.alert)
+}
```

Comment:
Why does this differ from the previous code before re-ordering?

Reply:
should be fixed now